jiant/tasks/lib/mnli.py

```diff
@@ -30,6 +30,11 @@ def tokenize(self, tokenizer):
             label_id=MnliTask.LABEL_TO_ID[self.label],
         )
 
+    def get_input_a(self):
+        return self.premise
+
+    def get_input_b(self):
+        return self.hypothesis
```

Comment:
Useful to abstract `premise` and `hypothesis` to generalize to all double input tasks.

Reply:
I don't think we should put this code in the Task definition, since its primary purpose is to serve test code.

More broadly, if we may want to move toward streamlining the common task archetypes with e.g. templates like the multiple choice templates, in which case 1. we would standardize the fields and 2. we could also cluster tests based on that format. We should discuss this.

Reply:
100% agree with the move to streamline task types. However, my goal is to take a small step towards handling tasks in an abstract way (using `input_a` and `input_b`) to create a generic test for classification tasks. Then, once we refactor, we can run the test and ensure we have the same functionality. I'll set up a meeting to discuss :)